src/templates/blog-list.ts

```diff
@@ -24,60 +27,64 @@ const BlogIndex: FC<Props> = ({ data, location }) => {
       title: "All posts",
     }),
     posts.map(({ node }: { node: any }) => {
-      const title = node.frontmatter.title || node.fields.slug;
-      return h(
-        "article",
+      const {
+        fields: { slug },
+        frontmatter: { date, description, title },
+      } = node;
+      return h(ArticlePreview, {
+        key: slug,
+        slug,
+        title: title || slug,
+        date,
+        description,
+      });
+    }),
+    h(
+      "nav",
+      null,
+      h(
+        "ul",
         {
-          key: node.fields.slug,
           style: {
-            marginBottom: rhythm(1),
+            display: `flex`,
+            flexWrap: `wrap`,
+            justifyContent: `space-between`,
+            listStyle: `none`,
+            padding: 0,
           },
         },
         h(
-          "header",
+          "li",
           null,
-          h(
-            Typography,
-            {
-              variant: "h5",
-            },
+          previousPath &&
             h(
               Link,
               {
-                style: {
-                  boxShadow: `none`,
-                  textDecoration: "none",
-                },
-                to: node.fields.slug,
+                to: previousPath,
+                rel: "prev",
               },
-              title,
+              "\u2190 newer",
             ),
-          ),
-          h(
-            "small",
-            {
-              style: {
-                marginBottom: rhythm(1 / 4),
-              },
-            },
-            node.frontmatter.date,
-          ),
         ),
         h(
-          "section",
+          "li",
           null,
-          h(Typography, {
-            dangerouslySetInnerHTML: {
-              __html: node.frontmatter.description || node.excerpt,
-            },
-          }),
+          nextPath &&
+            h(
+              Link,
+              {
+                to: nextPath,
+                rel: "next",
+              },
+              "older \u2192",
```

Comment:
should `\u2192` have a readable variable name?

Reply:
I'll be a hip young dev and put the unicode character itself in, unescaped.